src/main/java/com/redhat/devtools/lsp4ij/ServerMessageHandler.java

```diff
@@ -54,7 +57,7 @@ private static NotificationType messageTypeToNotificationType(MessageType type)
 
 
     public static void showMessage(String title, MessageParams params) {
-        Notification notification = new Notification("Language Server Protocol", title, params.getMessage(), messageTypeToNotificationType(params.getType()));
+        Notification notification = new Notification("Language Server Protocol", title, toHTML(params.getMessage()), messageTypeToNotificationType(params.getType()), NotificationListener.URL_OPENING_LISTENER);
```

Comment:
Use bundle message for `Language Server Protocol`

Reply:
done